### PR TITLE
feat: import and start service from hugging face

### PR DESCRIPTION
**Pull request by agatha197**
### Import from Hugging Face Modal Enhancements

This PR implements the functionality to import models from Hugging Face and create model store folders or services. Key changes include:

- Added `importAndStartService` mutation to handle the import process
- Implemented success and error handling for the import operation
- Added notifications for the successful creation of model store folders and services
- Updated translations for new UI elements and messages

**Checklist:**

These changes allow users to easily import models from Hugging Face, creating either a model store folder or a new service. The UI provides feedback on the success of the operation and offers quick navigation to the newly created resources.

**Screenshots:**

### CODE REVIEW

**Comment by agatha197** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2713?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2713** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2713?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2710** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2710?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2709** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2709?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2514** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/ImportFromHuggingFaceModal.tsx [hunk 1]
```diff
@@ -132,7 +158,41 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
...
+                  to: `/serving/${data.service.endpoint_id}`,
+                });
+              }
+            },
+            onError(e) {
+              message.error(e.message || t('dialog.ErrorOccurred'));
+            },
+          },
+        );
         onRequestClose();
```
Comment (yomybaby) [inline react/src/components/ImportFromHuggingFaceModal.tsx]: Please handle `onRequestClose` after the mutation is finished.
Also please set the  submit button's loading prop to mutation's `isPending`

**Review by yomybaby — CHANGES_REQUESTED** [review]
Opening the folder for Notification is only available when the current WebUI project is "model-store", otherwise it will throw an error. The reason is that the imported folder is a  "project" folder of "model-store" project.

File: react/src/components/ImportFromHuggingFaceModal.tsx [hunk 2]
```diff
@@ -132,7 +158,41 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
...
+            importOnly: isImportOnly,
+            serviceName: values.service_name || undefined,
+            folderName: values.folder_name || undefined,
+          },
+          {
+            onSuccess(data: any) {
+              if (data?.folder?.id && data?.folder?.name) {
+                upsertNotification({
+                  key: 'ImportFromHuggingFaceModal' + data.folder.id,
+                  message: `${t('data.modelStore.NewModelStoreFolderHasBeenCreated')}: ${data.folder.name}`,
```
Comment (yomybaby) [inline react/src/components/ImportFromHuggingFaceModal.tsx]: You can use `description` too.



```suggestion
                  message: `${t('data.modelStore.NewModelStoreFolderHasBeenCreated')}`,
                  description: data.folder.name,
```

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by agatha197 — COMMENTED** [review]
handled with https://github.com/lablup/backend.ai-webui/pull/2717. close this issue.

**Pull request closed without merging**